ocs_ci/framework/conf/default_config.yaml

```diff
@@ -48,4 +48,4 @@ ENV_DATA:
   # cause we don't need to update it each time new 14.x version is released
   # but only once when move to new version like v15.
   ceph_image: 'ceph/daemon-base:latest-nautilus-devel'
-  rook_image: 'rook/ceph:v1.0.0-256.g01ba50e'
+  rook_image: 'rook/ceph:v0.9.0-519.g111610e'
```

Comment:
Still I'm not sure, why the `rook_image` is changed in this PR? (here and on the other places)

Reply:
I'll wait for the PR that fixing the version back to 'master' and rebase.
With that said, @ebenahar raised a good point about a core issue in this implementation I need to fix